docs/tempo/website/troubleshooting/_index.md

```diff
@@ -10,3 +10,4 @@ The topics in this section helps with day zero operational issues that may come
 - [I am unable to find my traces in Tempo](unable-to-see-trace)
 - [I am getting error message "Too many jobs in the queue"](too-many-jobs-in-queue)
 - [Spans are being refused with "pusher failed to consume trace data"](max-trace-limit-reached)
+- [Is the Grafana Agent sending to Grafana Cloud](agent)
```

Comment:
nit: should we make this specific to Grafana Cloud or `... sending to the Backend`?